tests/run-make/lto-avoid-object-duplication/rmake.rs

```diff
@@ -0,0 +1,38 @@
+// ignore-tidy-tab
+// Staticlibs don't include Rust object files from upstream crates if the same
+// code was already pulled into the lib via LTO. However, the bug described in
+// https://github.com/rust-lang/rust/issues/64153 lead to this exclusion not
+// working properly if the upstream crate was compiled with an explicit filename
+// (via `-o`).
+
+// This test makes sure that functions defined in the upstream crates do not
+// appear twice in the final staticlib when listing all the symbols from it.
+
+//@ ignore-windows
+// Reason: `llvm-objdump`'s output looks different on windows than on other platforms.
+// Only checking on Unix platforms should suffice.
```

Comment:
Remark (non-blocking): I'm not super convinced by this reasoning and it would be good to also make this test work on Windows in future improvements.

Reply:
I have temporarily removed the ignore directive, maybe we could `bors try` it.